code-challenges.js

```diff
@@ -29,8 +56,22 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
-
-
+// I need to create a function that orders all numbers in the array from least to greatest
+const leastToGreatest = (array) => {
+  array.sort((a,b) => a-b)
+  //now that the numbers are least to greatest, I want to return ONLY the lowest number which would be at index [0] and I want to return the last index of the numbers with array[array.length-1]
+  return array = [array[0], array[array.length-1]]
+}
```

Comment:
cool! I like this solution. but if you can take out `array =` out of the return. it's redundant because you're just creating a variable for the sake of returning it. you can just return `[array[0], array[array.length-1]]`